pkg/skaffold/build/scheduler_test.go

```diff
@@ -291,39 +292,73 @@ func TestInParallelConcurrency(t *testing.T) {
 			}
 
 			initializeEvents()
-			results, err := InParallel(context.Background(), ioutil.Discard, tags, artifacts, builder, test.limit)
+			results, err := InOrder(context.Background(), ioutil.Discard, tags, artifacts, builder, test.limit)
 
 			t.CheckNoError(err)
 			t.CheckDeepEqual(test.artifacts, len(results))
 		})
 	}
 }
 
-func TestInParallelForArgs(t *testing.T) {
+func TestInOrderForArgs(t *testing.T) {
```

Comment:
Need to have a test where a dependency build fails.

Reply:
done